game/world/world.cpp

```diff
@@ -117,6 +117,22 @@ void World::insertPlayer(std::unique_ptr<Npc> &&npc, std::string_view waypoint)
     game.script()->setInstanceNPC("HERO",*npcPlayer);
   }
 
+void World::setPlayer(Npc* npc) {
+  if(npc==nullptr)
+    return;
+  npcPlayer->setProcessPolicy(Npc::ProcessPolicy::AiNormal);
+  npcPlayer->resumeAiRoutine();
```

Comment:
That is still deferred, now - just wrapped :)
You can have design, like:
```c++
void Npc::resumeAiRoutine(){ /* actual work */ }
```
and
```c++
case AI_ContinueRoutine:
  resumeAiRoutine();
  break;
```

Reply:
Thanks for your patience and help :)